app/gui2/src/providers/functionInfo.ts

```diff
@@ -3,7 +3,10 @@ import type { AstId } from '@/util/ast/abstract.ts'
 import { identity } from '@vueuse/core'
 
 interface FunctionInfo {
+  /** Id of top-level application expression. */
   callId: AstId | undefined
+  /** Ids of all nested prefix applications inside top-level expression. */
+  prefixCalls: AstId[]
```

Comment:
As per previous comments, I believe we no longer need a separate `callId`. Instead, it could be part of `prefixCalls` (which should be a `Set<AstId>`).